boot/mynewt/src/main.c

```diff
@@ -207,8 +213,10 @@ int flash_device_base(uint8_t fd_id, uintptr_t *ret)
 int
 main(void)
 {
+#if !MYNEWT_VAL(BOOT_CUSTOM_START)
```

Comment:
let's just use rsp and flash_base here in the custom call as is.

Reply:
Fixed